docs/src/app/components/Master.js

```diff
@@ -22,15 +23,18 @@ class Master extends Component {
 
   static childContextTypes = {
     muiTheme: PropTypes.object,
+    uniqueIdGen: PropTypes.func.isRequired,
   };
 
   state = {
     navDrawerOpen: false,
+    uniqueIdGen: getUniqueIdGenerator(),
```

Comment:
I'm just following the pattern of `MuiThemeProvider`. The `uniqueIdGen` should be stored inside a component for server rendering(#3533). Though docs site doesn't support server rendering currently. 😂